CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Contributing to Aragon DAO Kits and Templates
```

Comment:
Boilerplate language that I'm happy to update based on the discussion. Wanted to get a better picture of the future before trying to change too much.

Reply:
I think this looks good